web/app/scripts/views/dashboard/dashboard-partial.html

```diff
@@ -1,13 +1,136 @@
-<div class="row">
-    <div class="form-area">
-        <div class="col-sm-8 col-sm-offset-3">
-            <div class="form-area-heading">
-                <h2>Dashboard page placeholder</h2>
-            </div>
-            <div driver-stepwise chart-data="ctl.records.results"></div>
-            <div driver-toddow chart-data="ctl.records.results"></div>
-            <div class="form-area-body">
-            </div>
-        </div>
-    </div>
+<main class="dashboard">
```

Comment:
I imagine we'll probably end up using some of it afterwards though when fleshing out the dashboard. I wasn't thinking anything excessive, just having your editor auto-format the tags.